src/js/components/DateInput/DateInput.js

```diff
@@ -262,19 +304,23 @@ Use the icon prop instead.`,
         <Keyboard
           onEsc={open ? () => closeCalendar() : undefined}
           onSpace={(event) => {
-            event.preventDefault();
-            openCalendar();
+            if (!readOnlyCopy) {
+              event.preventDefault();
+              if (!readOnly) openCalendar();
+            }
           }}
         >
-          <Box
+          <StyledDateInputContainer
             ref={containerRef}
             border={!plain}
             round={theme.dateInput.container.round}
             direction="row"
+            readOnlyProp={readOnly}
             fill
           >
-            {reverse && calendarButton}
+            {reverse && (!readOnly || readOnlyCopy) && calendarButton}
             <MaskedInput
+              readOnly={readOnly}
```

Comment:
Are we expecting people to pass `readOnly` and `readOnlyCopy` if they want the copy behavior? Or if they say `readOnlyCopy` should we also enforce `readOnly` for them?

Reply:
I was kinda also thinking the same I felt like if you had `readOnlyCopy` it should trigger `ReadOnly`

Reply:
> Or if they say readOnlyCopy should we also enforce readOnly for them?

Good idea, that would make things simpler. I'll make that change

Reply:
done